consensus/propagation/reactor.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	maxMsgSize = 4194304 // 4MiB
+	maxMsgSize = 512 * 1024 // 512kb
```

Comment:
the largest message the propagation reactor can receive is a compact block which is ~315kb. so 512kb is plenty

Reply:
most likely yes, but decided to set it also to 512kb to be on the safe size. I ran a network for sometime and it didn't show any issue